### PR TITLE
Improve docs for loading fonts locally

### PR DESCRIPTION
**Pull request by domyen**
Issue: na

## What I did
Improve docs for loading fonts from a static directory. This was a stumbling block for a [user in Discord](https://discord.com/channels/486522875931656193/501692020226654208/789646850008285195). 

## How to test

- Is this testable with Jest or Chromatic screenshots? no
- Does this need a new example in the kitchen sink apps? no
- Does this need an update to the documentation? no

### CODE REVIEW

File: docs/get-started/setup.md [hunk 1]
```diff
@@ -97,14 +97,21 @@ If you have global imported styles, create a file called [`.storybook/preview.js
...
+  <summary>Load fonts or images from a local directory</summary>
+
+If you're referencing fonts or images from a local directory, you'll need to configure the Storybook script to [serve the static files](../configure/images-and-assets).
 
 </details>
 
 ## Load assets and resources
 
-If you want to link to static files in your project or stories (e.g. `/fonts/XYZ.woff`), use the `-s path/to/folder` to specify a static folder to serve from when you start up Storybook. To do so, edit the `storybook` and `build-storybook` scripts in `package.json`.
+If you want to [link to static files](../configure/images-and-assets) in your project or stories (e.g. `/fonts/XYZ.woff`), use the `-s path/to/folder` to specify a static folder to serve from when you start up Storybook. To do so, edit the `storybook` and `build-storybook` scripts in `package.json`.
```
Comment (jonniebigodes) [inline docs/get-started/setup.md:115]: @domyen adjust this "../configure/images-and-assets " to "../configure/images-and-assets.md"

**Review by jonniebigodes — CHANGES_REQUESTED** [review]
@domyen one small adjustment and this should be good to go.

File: docs/get-started/setup.md (hunk 1 above)
Comment (domyen) [inline docs/get-started/setup.md:115]:
```suggestion
If you want to [link to static files](../configure/images-and-assets.md) in your project or stories (e.g. `/fonts/XYZ.woff`), use the `-s path/to/folder` to specify a static folder to serve from when you start up Storybook. To do so, edit the `storybook` and `build-storybook` scripts in `package.json`.
```

**Comment by domyen** [comment]
ty - made the change @jonniebigodes

File: docs/configure/images-and-assets.md [hunk 2]
```diff
@@ -22,7 +22,7 @@ Afterwards you can use any asset in your stories:
 
 ### Serving static files via Storybook
 
-We recommend serving static files via Storybook to ensure that your components always have the assets they need to load.
+We recommend serving static files via Storybook to ensure that your components always have the assets they need to load. This technique is recommended for assets that your component's often use like logos, fonts, and icons.
```
Comment (shilman) [inline docs/configure/images-and-assets.md:25]:
```suggestion
We recommend serving static files via Storybook to ensure that your components always have the assets they need to load. This technique is recommended for assets that your components often use like logos, fonts, and icons.
```

**Review by shilman — APPROVED** [review]

**Pull request merged**